dependency updates report generation in xml does not create target directory

On clean project:
```
[master|⚑ 11] $ mvn org.codehaus.mojo:versions-maven-plugin:2.5-SNAPSHOT:dependency-updates-report -DdependencyUpdateReportFormats=xml
[INFO] Scanning for projects...
[INFO] 
...
[INFO] BUILD FAILURE
...
[ERROR] Failed to execute goal org.codehaus.mojo:versions-maven-plugin:2.5-SNAPSHOT:dependency-updates-report (default-cli) on project geo-calc-job: An error has occurred in Dependency Updates Report report generation.: Cannot create xml report. /home/idubinin/git/geo-calc/job/target/dependency-updates-report.xml (No such file or directory) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```
But when target directory exist:
```
[master|⚑ 11] $ mkdir target
  [master|⚑ 11] $ mvn org.codehaus.mojo:versions-maven-plugin:2.5-SNAPSHOT:dependency-updates-report -DdependencyUpdateReportFormats=xml
[INFO] Scanning for projects...
[INFO] 
[INFO] ------------------------------------------------------------------------
...
[INFO] 
[INFO] --- versions-maven-plugin:2.5-SNAPSHOT:dependency-updates-report (default-cli) @ geo-calc-job ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1.883 s
[INFO] Finished at: 2017-07-13T15:04:11+01:00
[INFO] Final Memory: 19M/323M
[INFO] ------------------------------------------------------------------------

```

--- a/src/main/java/org/codehaus/mojo/versions/DependencyUpdatesReport.java
+++ b/src/main/java/org/codehaus/mojo/versions/DependencyUpdatesReport.java
@@ -165,8 +165,13 @@ public class DependencyUpdatesReport
                 }
                 else if ( "xml".equals( format ) )
                 {
+                   File outputDir = new File(getProject().getBuild().getDirectory());
+                   if (!outputDir.exists())
+                    {
+                        outputDir.mkdirs();
+                    }
                     String outputFile =
-                        getProject().getBuild().getDirectory() + File.separator + getOutputName() + ".xml";
+                        outputDir.getAbsolutePath() + File.separator + getOutputName() + ".xml";
                     DependencyUpdatesXmlRenderer xmlGenerator =
                         new DependencyUpdatesXmlRenderer( dependencyUpdates, dependencyManagementUpdates, outputFile );
                     xmlGenerator.render();

--- a/src/main/java/org/codehaus/mojo/versions/PluginUpdatesReport.java
+++ b/src/main/java/org/codehaus/mojo/versions/PluginUpdatesReport.java
@@ -122,8 +122,13 @@ public class PluginUpdatesReport
                 }
                 else if ( "xml".equals( format ) )
                 {
+                    File outputDir = new File(getProject().getBuild().getDirectory());
+                    if (!outputDir.exists())
+                    {
+                        outputDir.mkdirs();
+                    }
                     String outputFile =
-                        getProject().getBuild().getDirectory() + File.separator + getOutputName() + ".xml";
+                        outputDir.getAbsolutePath() + File.separator + getOutputName() + ".xml";
                     PluginUpdatesXmlRenderer xmlGenerator =
                         new PluginUpdatesXmlRenderer( pluginUpdates, pluginManagementUpdates, outputFile );
                     xmlGenerator.render();

